crypto/hmac/hmac_s390x.c

```diff
@@ -263,6 +263,7 @@ int s390x_HMAC_CTX_copy(HMAC_CTX *dctx, HMAC_CTX *sctx)
     memcpy(&dctx->plat.s390x.param, &sctx->plat.s390x.param,
            sizeof(dctx->plat.s390x.param));
 
+    OPENSSL_clear_free(dctx->plat.s390x.buf, dctx->plat.s390x.size);
     dctx->plat.s390x.buf = NULL;
```

Comment:
Could also clear the size here.. But looks like it wont have any side effects.

Reply:
The size is set below. In case `OPENSSL_memdup()` fails before setting the size, `s390x_HMAC_CTX_copy()` returns 0 which in turn causes the caller (`HMAC_CTX_copy()`) to call `hmac_ctx_cleanup()` which calls `s390x_HMAC_CTX_cleanup()` which sets the size to zero. So we always end up with either `buf` being NULL and `size` being zero, or with a valid `buf` pointer and a valid `size`.